app/(components)/navbar/navbar.module.css

```diff
@@ -0,0 +1,66 @@
+.nav {
+  height: 4rem;
+  display: flex;
```

Comment:
Added a [task to setup](https://github.com/orgs/ed-pilots-network/projects/4/views/1?pane=issue&itemId=30752047) some standards around CSS usage